source/qb64pe.bas

```diff
@@ -3685,6 +3696,7 @@ DO
             'check for further occurrences
             spo = INSTR(spo + 2, a3u$, qb64prefix$ + "EMBEDDED$")
```

Comment:
I think you need to add the same check you do before loop again down here. Lines like `Print _Embedded$("foo"); var_embedded$` trigger the check on the `var_embedded$` when they shouldn't.